go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/markbates/goth
+module github.com/akramer/goth
```

Comment:
Looks like this change was probably added by accident when you worked on your fork – could you remove this?